api/src/exam-environment/routes/exam-environment.test.ts

```diff
@@ -0,0 +1,38 @@
+import { examId } from '../../../__mocks__/new-exam';
+import {
+  createSuperRequest,
+  devLogin,
+  seedNewExam,
+  setupServer
+} from '../../../jest.utils';
+
+describe('/exam-environment', () => {
+  setupServer();
+  describe('Authenticated user', () => {
+    let setCookies: string[];
+    let superPost: ReturnType<typeof createSuperRequest>;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    let examEnvironmentAuthorizationToken: string;
+
+    // Authenticate user
+    beforeAll(async () => {
+      setCookies = await devLogin();
+      superPost = createSuperRequest({ method: 'POST', setCookies });
+      await seedNewExam();
+      // Add exam environment authorization token
+      const res = await superPost('/user/exam-environment/token');
+      expect(res.status).toBe(200);
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      examEnvironmentAuthorizationToken =
```

Comment:
This is a WIP?

Reply:
Yes, ignore the tests. I stopped writing tests when I realised I was changing the schema twice per week.